config/rewrites.d/developer.rackspace.com.json

```diff
@@ -322,14 +322,14 @@
         {
           "description": "Redirect unversioned RPC URL to current version",
           "from": "^\\/docs\\/private-cloud\\/rpc\\/?$",
-          "to": "/docs/private-cloud/rpc/v14",
+          "to": "/docs/private-cloud/rpc/v16",
           "rewrite:": false,
           "status": 301
         },
         {
-          "description": "Redirect older RPC URLs to current version",
-          "from": "^\\/docs\\/private-cloud\\/rpc\\/\bv(9|1[01])\b\\/?(.*)$",
-          "to": "/docs/private-cloud/rpc/v14",
+          "description": "Redirect older RPC URLs to current version minus one",
+          "from": "^\\/docs\\/private-cloud\\/rpc\\/\bv(9|1[0123])\b\\/?(.*)$",
```

Comment:
we still want v12 and v13 docs available to customers. will this line [0123] cause problems?

Reply:
They're available, just redirected. Did you not want a redirect?

Reply:
sorry, don't follow you. 
If a customer types i `https://developer.rackspace.com/docs/private-cloud/rpc/v11/rpc-releasenotes` we want to redirect them, since those docs are retired.
If a customer types in `https://developer.rackspace.com/docs/private-cloud/rpc/v12/rpc-releasenotes` we *don't * want them redirected to v16, since those docs are still available.

Reply:
got it; I'll update